scripts/do.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# This script is an interface to any of the methods of lib.sh
+# Call this script as "do.sh method_from_lib" to execute any function from that library
+set -eu
+
+HERE="$(cd "$(dirname "${BASH_SOURCE[0]}")"; pwd -P)"
+# shellcheck disable=SC1090
+source "${HERE}/lib.sh"
+
+# This block is to enable reporting issues to Sentry.io
+if ! require sentry-cli; then
+    curl -sL https://sentry.io/get-cli/ | bash
+fi
```

Comment:
Should this only be installed when below conditions with the env vars is true?

Reply:
I had thought about it, however, only 1 call w/o the variable in place would get it installed.
I could add it if you prefer me to, however, there are no negative consequences if it gets installed.